src/hooks/api.ts

```diff
@@ -2,12 +2,6 @@ import { useEffect, useState } from 'react'
 
 /* eslint-disable camelcase */
```

Comment:
Cool thanks Memoyil, you can probably remove this now too

Reply:
Done